jinja2/includes/config.html

```diff
@@ -5,20 +5,23 @@
     (function(win) {
         'use strict';
 
-        {{ waffle.wafflejs() }}
-
+        {#- This is stuff not needed by the React-based site #}
         {%- if not beta %}
-        // This needs to be set before ckeditor.js loads
-        {%- if settings.CKEDITOR_DEV %}
-          {%- set ckeditor_basepath = static('js/libs/ckeditor4/source/ckeditor/') %}
-        {%- else %}
-          {%- set ckeditor_basepath = static('js/libs/ckeditor4/build/ckeditor/') %}
-        {%- endif %}
-        window.CKEDITOR_BASEPATH = '{{ ckeditor_basepath }}';
+            {#- With React, waffle flags are returned by the /whoami API. #}
+            {{ waffle.wafflejs() }}
```

Comment:
This waffle.wafflejs() used to be outside the if not beta block. Now it is inside, since we don't want to cache waffle flags. Note that the changed lines below are just indentation changes.